mattermost-plugin/server/plugin.go

```diff
@@ -52,6 +53,8 @@ func (p *Plugin) OnActivate() error {
 		return fmt.Errorf("cannot activate plugin: %w", err)
 	}
 
+	model.LogServerInfo(logger)
```

Comment:
@mgdelacroix can you add this to product mode as well? (https://github.com/mattermost/focalboard/blob/feb49eaf197afed8c082735837ebf95e04210621/mattermost-plugin/product/boards_product.go#L200)

Reply:
Done on [b087f27](https://github.com/mattermost/focalboard/pull/3711/commits/b087f27ddf8277471c8405f296b5a42d8c732969), please let me know if it's the right place to add it

Reply:
Oups, just saw that the comment had a line included, quickly fixing

Reply:
Done on 4d5d16a, now log happens right after the `New` call